### PR TITLE
Escrow Migration logic back

### PR DESCRIPTION
**Pull request by clementbalestrat**

### CODE REVIEW

File: queries/escrow/useEscrowDataQuery.tsx [hunk 1]
```diff
@@ -106,6 +112,7 @@ const useEscrowDataQuery = (options?: QueryConfig<EscrowData>) => {
 				totalVested: totalVested / 1e18,
 				claimableEntryIds,
 				claimableEntryIdsInChunk,
+				totalBalancePendingMigration: totalBalancePendingMigration / 1e18,
```
Comment (andytcf) [inline queries/escrow/useEscrowDataQuery.tsx:115]: @clementbalestrat it'll be cool if we refactored it to be like

```
const [claimableAmount, totalEscrowed, totalVested, totalBalancePendingMigration] = [
				claimableAmountBN,
				totalEscrowedBN,
				totalVestedBN,
				totalBalancePendingMigrationBN,
			].map((data) => Number(synthetix.js?.utils.formatEther(data)));
```

File: sections/escrow/components/StakingRewardsTab/StakingRewardsTab.tsx [hunk 2]
```diff
@@ -112,7 +110,9 @@ const StakingRewardsTab: React.FC = () => {
 						txHash: transaction.hash,
 						onTxConfirmed: () => {
 							setTransactionState(Transaction.SUCCESS);
-							escrowDataQuery.refetch();
+							setTimeout(() => {
```
Comment (andytcf) [inline sections/escrow/components/StakingRewardsTab/StakingRewardsTab.tsx:113]: maybe refactor to utils call it, delayedRefetch if this is a common pattern

**Review by andytcf — CHANGES_REQUESTED** [review]
left some comments, nice work

File: sections/escrow/components/StakingRewardsTab/StakingRewardsTab.tsx (hunk 2 above)
Comment (clementbalestrat) [reply sections/escrow/components/StakingRewardsTab/StakingRewardsTab.tsx:113]: Yeah I was thinking about this. Let's do it later tho

**Review by andytcf — APPROVED** [review]

**Pull request merged**